manifests/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -245,9 +247,11 @@ spec:
           - name: sds-uds-path
             mountPath: /var/run/sds
             readOnly: true
+          {{- if eq .Values.global.jwtPolicy "third-party-jwt" }}
```

Comment:
you need to remove the volume too, not just the volume mount

Reply:
Added the conditional flag to the volume too. The mounting part will be addressed in follow up PRs.